### DISCUSSION TITLE
Why is there no link to my personal main page from the dashboard?

### DISCUSSION THREAD

**Discussion by Plasmadog** [community/community#32474] [Other Feature Feedback, Questions, & Ideas] [answered]
When I am on the landing page/dashboard and I select one of my organizations as the context, I see a "view organization" button that takes me to the main page for that organization. But when I choose my own personal account as the context, there is no equivalent button. There does not appear to be any way to get to my own main page except by typing the url. This would not be so bad if it were not for the fact that the list of recent repositories is mostly made up of repos that are not mine, but as it is there is no easy way to get to a list of _just my own_ repos. Maybe the personal context is useful to someone who works with multiple organizations, but for me it just obstructs what I actually want to see.

**Comment by Vladimir-Moskalenko** [comment, marked as answer]
Maybe you should try doing this using the top-right corner with your avatar? In the popup menu select 'Signed in as <your username>'.

**Reply by Plasmadog** [reply]
Wow, that is not obvious. Thanks Vladimir.

**Comment by jfullstackdev** [comment]
can you have the screenshot of that ? 

this is what you should be seeing

![image](https://user-images.githubusercontent.com/47092464/189491688-8ce9ac22-ba9a-400e-af91-e34b23a4e992.png)
